src/azul/service/manifest_service.py

```diff
@@ -878,7 +884,11 @@ def _get_related_rows(self, doc: dict, row: dict) -> Iterable[dict]:
         file_ = one(doc['contents']['files'])
         for related in file_['related_files']:
             new_row = row.copy()
-            new_row.update({'file_' + k: v for k, v in related.items()})
+            for field_name, field_value in related.items():
+                if field_name == 'drs_path':
+                    new_row['file_drs_uri'] = self.repository_plugin.drs_uri(field_value)
```

Comment:
This is a brittle coupling with with https://github.com/DataBiosphere/azul/blob/a6040017c02863547d3da52c237f5a8c0bcaf424/src/azul/plugins/metadata/hca/__init__.py#L202

For normal files this is handled differently.